pkg/fanal/secret/testdata/azure-secrets.txt

```diff
@@ -0,0 +1,6 @@
+"value": "JpN8Q~Q~qX_ENrlfqR.7_2-OqNVKdRG71Vbupaf~"
```

Comment:
Is this file used somewhere?

Reply:
Just test secrets generated in Azure, not used anywhere 👍 Thanks

Reply:
Just test secrets generated in Azure, not used anywhere 👍 Thanks

Reply:
Could you add a test case for this file then?

Reply:
Added a secret scan validation test workflow that can validate supported secret types. When someone adds a built-in rule they can put the secret test data in trivy/pkg/fanal/secret/testdata

Reply:
Added a secret scan validation test workflow that can validate supported secret types. When anyone adds a built-in rule they can put test data in pkg/fanal/secret/testdata

Reply:
I meant unit tests here.
https://github.com/aquasecurity/trivy/blob/be97daa2418891eb39b5164f1187757bcac60b04/pkg/fanal/secret/scanner_test.go

This test looks too much, by the way. We can include one or two secrets in the test data.